server/src/main/java/org/eclipse/lsp/cobol/core/model/tree/ParagraphDefinitionNameNode.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (c) 2022 Broadcom.
+ * The term "Broadcom" refers to Broadcom Inc. and/or its subsidiaries.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *    Broadcom, Inc. - initial API and implementation
+ *
+ */
+package org.eclipse.lsp.cobol.core.model.tree;
+
+import com.google.common.collect.ImmutableList;
+import lombok.Getter;
+import lombok.Setter;
+import org.eclipse.lsp.cobol.core.model.Locality;
+import org.eclipse.lsp.cobol.core.model.SyntaxError;
+import org.eclipse.lsp4j.Location;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/** The class represents paragraphs name node in COBOL grammar. */
+@Getter
+public class ParagraphDefinitionNameNode extends Node implements Context {
+  private final String paragraphName;
+  @Setter List<Location> definitions = new ArrayList<>();
```

Comment:
Why it is a list? There may be only one definition.

Reply:
But I don't think that you need a setter here. Can you use `addDefinition` instead? Like you did for the usages.

Reply:
I removed setter, But I am not sure if we should throw any error for multiple definitions. I checked in with existing extension and it does not throw any error.